Presentation.Web/app/models/ViewModel/it-system-usage/system-usage-hostedat-view-model.ts

```diff
@@ -0,0 +1,25 @@
+﻿module Kitos.Models.ViewModel.ItSystemUsage {
+
+    export interface IHostedAtModel {
+        textValue: string;
+        text: string;
+    }
+
+    export class HostedAtViewModel {
+        static readonly hostedAtOptions = {
+            UNDECIDED: <IHostedAtModel>{ textValue: "UNDECIDED", text: "" },
+            ONPREMISE: <IHostedAtModel>{ textValue: "ONPREMISE", text: "On-premise" },
+            EXTERNAL: <IHostedAtModel>{ textValue: "EXTERNAL", text: "Eksternt" },
+        };
+
+        static getTextValueToTextMap() {
```

Comment:
Den skal ikke beregnes for hver gang man spørger men gemmes som en static readonly som så kan tilgås via denne public metode. Ellers er der ikke megen mening i at have et map :-)